src/parse.c

```diff
@@ -2228,128 +2276,128 @@ static const mapping_entry_handler dhcp6_overrides_handlers[] = {
-    {"optional-addresses", YAML_SEQUENCE_NODE, handle_optional_addresses},                    \
-    {"renderer", YAML_SCALAR_NODE, handle_netdef_renderer},                                   \
-    {"routes", YAML_SEQUENCE_NODE, handle_routes},                                            \
-    {"routing-policy", YAML_SEQUENCE_NODE, handle_ip_rules}
+    {"accept-ra", YAML_SCALAR_NODE, {.generic=handle_accept_ra}, netdef_offset(accept_ra)},        \
+    {"activation-mode", YAML_SCALAR_NODE, {.generic=handle_activation_mode}, netdef_offset(activation_mode)}, \
+    {"addresses", YAML_SEQUENCE_NODE, {.generic=handle_addresses}},                                      \
+    {"critical", YAML_SCALAR_NODE, {.generic=handle_netdef_bool}, netdef_offset(critical)},        \
+    {"ignore-carrier", YAML_SCALAR_NODE, {.generic=handle_netdef_bool}, netdef_offset(ignore_carrier)}, \
+    {"dhcp4", YAML_SCALAR_NODE, {.generic=handle_netdef_bool}, netdef_offset(dhcp4)},              \
+    {"dhcp6", YAML_SCALAR_NODE, {.generic=handle_netdef_bool}, netdef_offset(dhcp6)},              \
+    {"dhcp-identifier", YAML_SCALAR_NODE, {.generic=handle_dhcp_identifier}},                            \
+    {"dhcp4-overrides", YAML_MAPPING_NODE, {.map={FALSE, {.handlers=dhcp4_overrides_handlers}}}},                   \
+    {"dhcp6-overrides", YAML_MAPPING_NODE, {.map={FALSE, {.handlers=dhcp6_overrides_handlers}}}},                   \
+    {"gateway4", YAML_SCALAR_NODE, {.generic=handle_gateway4}},                                          \
+    {"gateway6", YAML_SCALAR_NODE, {.generic=handle_gateway6}},                                          \
+    {"ipv6-address-generation", YAML_SCALAR_NODE, {.generic=handle_netdef_addrgen}},                     \
+    {"ipv6-address-token", YAML_SCALAR_NODE, {.generic=handle_netdef_addrtok}, netdef_offset(ip6_addr_gen_token)}, \
+    {"ipv6-mtu", YAML_SCALAR_NODE, {.generic=handle_netdef_guint}, netdef_offset(ipv6_mtubytes)},  \
+    {"ipv6-privacy", YAML_SCALAR_NODE, {.generic=handle_netdef_bool}, netdef_offset(ip6_privacy)}, \
+    {"link-local", YAML_SEQUENCE_NODE, {.generic=handle_link_local}},                                    \
+    {"macaddress", YAML_SCALAR_NODE, {.generic=handle_netdef_mac}, netdef_offset(set_mac)},        \
+    {"mtu", YAML_SCALAR_NODE, {.generic=handle_netdef_guint}, netdef_offset(mtubytes)},            \
+    {"nameservers", YAML_MAPPING_NODE, {.map={FALSE, {.handlers=nameservers_handlers}}}},                           \
+    {"optional", YAML_SCALAR_NODE, {.generic=handle_netdef_bool}, netdef_offset(optional)},        \
+    {"optional-addresses", YAML_SEQUENCE_NODE, {.generic=handle_optional_addresses}},                    \
+    {"renderer", YAML_SCALAR_NODE, {.generic=handle_netdef_renderer}},                                   \
+    {"routes", YAML_SEQUENCE_NODE, {.generic=handle_routes}},                                            \
```

Comment:
I ended up removing them here, as the fields are way too varying for it to render cleanly, but I've kept them in the other instance, since all fields have the same type, thus same approximate length.